framework/caching-cmm.md

```diff
@@ -30,11 +39,15 @@ in this document are to be interpreted as described in [RFC 2119](https://tools.
 
 ## Initialization
 
-On caching CMM initialization, a caching CMM MUST define the following:
+On caching CMM initialization,
+the caller MUST provide the following values:
 
 - [Underlying Cryptographic Materials Cache (CMC)](#underlying-cryptographic-materials-cache)
 - [Underlying Cryptographic Materials Manager (CMM)](#underlying-cryptographic-materials-manager)
 - [Cache Limit TTL](#cache-limit-ttl)
+
+Additionally, the caching CMM MUST define the following:
+
```

Comment:
Per our latest wording, these should be "the caching CMM MUST optionally accept"